src/components/navbar.js

```diff
@@ -46,279 +45,491 @@ import SendIcon from '@material-ui/icons/Send';
-        <Link className="homeNavLink" to="/Login">Sign In</Link>
-        <Link to="/register"><button className="Signup" >Sign Up</button></Link>
-
-
-      </div> :
-        isLoggedIn() ? (
-          <AppBar position="fixed" style={{ boxShadow: "none", backgroundColor: "#0F3088", height: "8vh" }} >
-            <Toolbar style={{ minHeight: "0", color: 'black' }}>
-              {/* <Typography variant="h3" className={classes.title} style={{border: "1px solid black", width: "400px"}}>
+	const classes = useStyles();
+	const [auth, setAuth] = React.useState(true);
+	const [anchorEl, setAnchorEl] = React.useState(null);
+	const open = Boolean(anchorEl);
+
+	const handleChange = event => {
+		setAuth(event.target.checked);
+	};
+
+	const handleMenu = event => {
+		setAnchorEl(event.currentTarget);
+	};
+
+	const handleClose = () => {
+		setAnchorEl(null);
+	};
+
+	function isLoggedIn() {
+		return (
+			props.loggedInUser.firstName &&
+			props.loggedInUser.lastName &&
+			localStorage.getItem('token') &&
+			localStorage.getItem('userID')
+		);
+	}
+
+	function businessSearch() {
+		props.history.push('/search/business');
+		console.log('businessSearch');
+	}
+
+	function settings() {
+		props.history.push('/settings');
+	}
+
+	function competitorSearch() {
+		props.history.push('/search/competitor');
+		console.log('compSearch');
+	}
+
+	const handleClick = event => {
+		//event.preventDefault()
+		localStorage.removeItem('token');
+		localStorage.removeItem('userID');
+		props.shouldUpdateLoggedInUser(true);
+	};
+
+	function isOnHomePage() {
+		//this function is neccessary because match.params will always be "/" even while on /dashboard since the nav bar is always rendered to "/" (path is unexact "/")
+
+		var url = window.location;
+		var firstParam = url.pathname.split('/')[1];
+
+		return firstParam === '';
+	}
+
+	function isOnDashboard() {
+		//this function is neccessary because match.params will always be "/" even while on /dashboard since the nav bar is always rendered to "/" (path is unexact "/")
+
+		var url = window.location;
+		var firstParam = url.pathname.split('/')[1];
+
+		return firstParam.toUpperCase() === 'Dashboard'.toUpperCase();
+	}
+
+	return (
+		<div className={classes.root} style={{ position: 'fixed' }}>
+			{isOnHomePage() ? (
+				<div className='landing-top-section'>
+					<Link className='homeNavLink' to='/About'>
+						About
+					</Link>
+					<Link className='homeNavLink' to='/Login'>
+						Sign In
+					</Link>
+					<Link to='/register'>
+						<button className='Signup'>Sign Up</button>
+					</Link>
+				</div>
+			) : isLoggedIn() ? (
+				<AppBar
+					position='fixed'
+					style={{
+						boxShadow: 'none',
+						backgroundColor: '#0F3088',
+						height: '8vh'
+					}}
+				>
+					<Toolbar style={{ minHeight: '0', color: 'black' }}>
+						{/* <Typography variant="h3" className={classes.title} style={{border: "1px solid black", width: "400px"}}>
```

Comment:
commented out code

Reply:
Agreed! I haven't looked into this component yet. It was VS code auto-updating the imports.